src/main_test.go

```diff
@@ -51,3 +51,11 @@ func TestJavaScriptInjection(t *testing.T) {
 		t.Fatal("JavaScript injection attack")
 	}
 }
+
+func TestConvertFoo(t *testing.T) {
+	path := "/foo/bar"
+	text := helloHtml(path)
+	if strings.Contains(text, "foo") {
+		t.Fatal("'foo' in path")
+	}
+}
```

Comment:
As far as I can tell, not a big deal?

Reply:
@jthomp13 Could you clarify? `Fatal` is equivalent to `Log` followed by `FailNow` – [here are the docs for that](https://golang.org/pkg/testing/#B.Fatal). It will basically just cause this function to end and the test harness for Go will go on to the next test.

Reply:
@ryanhofdotgov @jthomp13 is referring to the "no newline at end of file" message that appears on line 61.  I think its a GitHub thing, Go doesn't seem to care.

Reply:
Ah, OK – thanks. The little red symbol doesn't appear on the excerpt shown on this page.

I'd expect to see the newline normally, but depending on what editor you used or settings on your editor, it may not add it. For *nix text parsing tools like `grep`, `tr`, etc. consistently having newlines is helpful, but otherwise most programming language compilers/interpreters don't care. If you have developers on Windows as well as macOS or Linux, then sometimes line-endings need to be made consistent with a pre-commit hook, etc.

@jthomp13 I'll let you have the honor of clicking the Approve button. :)

Reply:
Sorry  for the confusion, I thought that my comment was pointing to the line. I'll go ahead and approve it.